airflow/providers/cncf/kubernetes/operators/job.py

```diff
@@ -366,10 +366,17 @@ class KubernetesDeleteJobOperator(BaseOperator):
     :param in_cluster: run kubernetes client with in_cluster configuration.
     :param cluster_context: context that points to kubernetes cluster.
         Ignored when in_cluster is True. If None, current-context is used. (templated)
+    :param on_status: Condition for performing delete operation depending on the job status. Values:
+        ``None`` - delete the job regardless of its status, "Complete" - delete only successfully completed
+        jobs, "Failed" - delete only failed jobs. (default: ``None``)
+    :param wait_until_job_complete: Whether to wait for the job to complete. (default: ``False``)
+    :param job_poll_interval: Interval in seconds between polling the job status. Used when the `on_status`
```

Comment:
Same as above from what i can tell most operators have convention of `poll_interval`